dev-docs/source/Testing/ErrorReporter-ProjectRecovery/ErrorReporterTesting.rst

```diff
@@ -28,78 +28,110 @@ Error Reporter test
 
-5. Open MantidWorkbench
+5. MantidWorkbench is still open
 
 - Load the file `Training_Exercise3a_SNS.nxs`
 - Run the `NormaliseToMonitor` algorithm
-- This should cause an error box saying Mantid has thrown an unexpected exception
-- Make sure the hyperlink to the privacy policy works
+- This should cause an error reporter dialog saying Mantid has thrown an unexpected exception
+- Contact details from before should NOT be displayed and the `Remember Me` checkbox NOT ticked
 - Enter some text in the `Name` box; make sure the `Share non-identifiable information` button gets greyed out
-- In the `Name` box enter `Public2` and in the `Email` box enter `public2`
+- Input some additional information into the main textbox. Try to include characters that need to be escaped such as `"`
+- Leave the `Name` box EMPTY and in the `Email` box enter `public2`
+- Tick the `Remember Me` checkbox
 - Make sure the `Continue` radio button is checked
-- Click the `Yes, share information` box
+- Click the `Yes, share information` button
 - You should be returned to the main Mantid window
-- Check with the database admin
+- Check with the database admin that an error report was sent WITH a name, email, stacktrace and a textbox.
 
 ---------------
 
-6. Open MantidWorkbench
+6. MantidWorkbench is still open
 
 - Load the file `Training_Exercise3a_SNS.nxs`
 - Run the `NormaliseToMonitor` algorithm
-- This should cause an error box saying Mantid has thrown an unexpected exception
-- Make sure the hyperlink to the privacy policy works
+- This should cause an error reporter dialog saying Mantid has thrown an unexpected exception
+- Only the `public2` email from Test 5 should be displayed in the email box and the `Remember Me` checkbox ticked.
+  Make sure the `Share non-identifiable information` button is currently enabled (not greyed out)
 - Enter some text in the `Name` box; make sure the `Share non-identifiable information` button gets greyed out
 - In the `Name` box enter `Public3` and in the `Email` box enter `public3`
+- Input some additional information into the main textbox. Try to include characters that need to be escaped such as `"`
+- Tick the `Remember Me` checkbox
 - Make sure the `Terminate` radio button is checked
-- Click the `Yes, share information` box
+- Click the `Yes, share information` button
 - Mantid should shut down
-- Check with the database admin
+- Check with the database admin that an error report was sent WITH a name, email, stacktrace and a textbox.
+
+--------------
+
+7. Open MantidWorkbench
+
+- Load the file `Training_Exercise3a_SNS.nxs`
+- Run the `NormaliseToMonitor` algorithm
+- This should cause an error reporter dialog saying Mantid has thrown an unexpected exception
+- The `Public3` contact details from Test 6 should be displayed and the `Remember Me` checkbox ticked
+
+Test the error reporter with any weird and wonderful ideas.
+Note any problems with Workbench or these testing instructions.
 
+**Thanks for testing!!!***
```

Comment:
There looks to be an extra asterisk here

Reply:
cheers!